articles/confidential-computing/faq.md

```diff
@@ -27,32 +27,35 @@ If your Azure issue is not addressed in this article, visit the Azure forums on
 
 1. **Will all OS images work with Azure confidential computing?**
 
-   No. The virtual machines can only be deployed on Generation 2 virtual machines. We offer Generation 2 support for Ubuntu Server 18.04, Ubuntu Server 16.04, and Windows Server 2016 Datacenter. Read more about Gen 2 VMs on [Linux](../virtual-machines/linux/generation-2.md) and [Windows](../virtual-machines/windows/generation-2.md)
+   No. The virtual machines can only be deployed on Generation 2 virtual machines with Ubuntu Server 18.04, Ubuntu Server 16.04, and Windows Server 2016 Datacenter. Read more about Gen 2 VMs on [Linux](../virtual-machines/linux/generation-2.md) and [Windows](../virtual-machines/windows/generation-2.md)
 
 1. **DCsv2 virtual machines are grayed out in the portal and I can't select one**
 
-   Based on the information bubble next to the VM, there are different actions to take:
+   Make sure to toggle the filters as listed below and see if they become available:
     -	**UnsupportedGeneration**: Change the generation of the virtual machine image to “Gen2”.
     -	**NotAvailableForSubscription** : The region isn't yet available for your subscription. Select an available region.
-    -	**InsufficientQuota**: [Create a support request to increase your quota](../azure-portal/supportability/per-vm-quota-requests.md). Free trial subscriptions don't have quota for confidential computing VMs. 
+    
```

Comment:
these are errors that you need to hover over an information bubble to see. Many users don't know to see the information bubble. These errors only appear if the user sees the machines grayed out. toggle the filters will not fix these issues. The question below this refers to toggling the filters correctly.